qiskit/transpiler/passes/calibration/builders.py

```diff
@@ -103,26 +102,29 @@ class RZXCalibrationBuilder(CalibrationBuilder):
     angle. Additional details can be found in https://arxiv.org/abs/2012.11660.
     """
 
-    def __init__(self, backend: basebackend):
+    def __init__(
+        self,
+        instruction_schedule_map: InstructionScheduleMap,
+        qubit_channel_mapping: List[List[str]],
+    ):
```

Comment:
Technically this is a public API breaking change. Are we ok to do this?  At the very least, we'll need an upgrade note.

Reply:
Has this pass been in a release? I looked at the git log and it showed the first commit at 08/31 which was after the 0.18 release

Reply:
Yes - it was in #5752 which was in 0.17.  Its import path and a few other things have subsequently changed, but the object has been publicly released.

Reply:
ah, ok that's what I get for not looking closely. Then let me change this PR to make the backend kwarg optional and deprecate it

Reply:
Ok, it's being deprecated first in: https://github.com/Qiskit/qiskit-terra/pull/7331/commits/5225120569bf80f22106f5d3e64160b1eb491d0d